resources.md

```diff
@@ -15,3 +15,4 @@ Resources including existing curriculum, workshop materials and lesson planning.
 - [Metamesh YouTube channel (Video)](https://www.youtube.com/channel/UCGEnntxbGKMU9J9GIZ1LQUQ)
 - [Detroit Community Technology Project](http://detroitcommunitytech.org/learning-materials)
 - [Community Technology Facilitation](http://csommunitytechnology.github.io/docs/facilitation/discussion-formats/)
+- [NETworkshop](https://criticalengineering.org/courses/networkshop/)
```

Comment:
@benhylau I'm not seeing and lesson plans or learning materials here or am I missing something? 
All the rest are direct links to material we can re-puspose and I think that is how we should think of "resources"

Reply:
I was referencing this just for their course focus high level structure. We don't need to include this in resources if it doesn't fit this list.